prov/gni/include/gnix_atomic.h

```diff
@@ -48,4 +48,3 @@ ssize_t _gnix_atomic(struct gnix_fid_ep *ep,
 		     uint64_t flags);
 
 #endif /* _GNIX_ATOMIC_H_ */
-
```

Comment:
This doesn't seem necessary.  And it will make the list of files changed shorter if you revert it.